lineid_plot.py

```diff
@@ -383,12 +389,14 @@ def plot_line_ids(wave, flux, line_wave, line_label1, label1_size=None,
                     fontsize=label1_size[i],
                     arrowprops=dict(arrowstyle="-",
                                     relpos=(0.5, 0.0)),
-                    label=label_u[i])
+                    label=label_u[i],
+                    **annotation_kwargs)
         if extend[i]:
             ax.plot([line_wave[i]] * 2, [arrow_tip[i], line_flux[i]],
-                    linestyle="--", color="k",
                     scalex=False, scaley=False,
-                    label=label_u[i] + "_line")
+                    label=label_u[i] + "_line",
+                    **plot_kwargs,
```

Comment:
Fixed.  Strange, my editor didn't catch this, and the code installed & worked just fine with this in there.  But I made a MWE and my editor stated it was a syntax error.  So I think it may not actually be invalid syntax, just ugly?